proto/reporting/input_params/management_report_manifests.proto

```diff
@@ -12,6 +12,13 @@ message ManagementCurrentTimeReportManifest {
     bool folio_balance_report = 2;
     bool lodging_verification_report = 3;
     bool comp_stay_report = 4;
+
+    bool prop_stats_yesterday = 5;
+    bool prop_stats_trailing_7d = 6;
+    bool prop_stats_trailing_28d = 7;
+    bool prop_stats_last_full_week = 8;
+    bool prop_stats_last_full_month = 9;
+    bool prop_stats_last_full_year = 10;
```

Comment:
Approved, but why do these have to be separate items on the manifest? Can this be parameterized some other way?